framework/db/ActiveRecord.php

```diff
@@ -772,7 +772,7 @@ protected function deleteInternal()
      * Returns a value indicating whether the given active record is the same as the current one.
      * The comparison is made by comparing the table names and the primary key values of the two active records.
      * If one of the records [[isNewRecord|is new]] they are also considered not equal.
-     * @param ActiveRecord $record record to compare to
+     * @param ActiveRecordInterface $record record to compare to
```

Comment:
That's not correct since there's `tableName()` call and it is not in `ActiveRecordInterface`.